packages/core/src/browser/keybinding.ts

```diff
@@ -288,25 +286,25 @@ export class KeybindingRegistry {
      * @param bindings the keybinding reference list
      * @param binding the keybinding to test collisions for
      */
-    containsKeybinding(bindings: Keybinding[], binding: Keybinding): boolean {
+    checkKeybindingCollisions(bindings: Keybinding[], binding: Keybinding): boolean {
```

Comment:
I think a `changelog` entry would be useful for this breaking change.

Reply:
I'D say there's no reason to rename this method as it's semantics haven't changed.

Reply:
Revered the method name back.